setup.py

```diff
@@ -139,7 +138,8 @@ def run(self):
         "ssh_gssapi": ssh_gssapi,
         "hdfs": hdfs,
         # NOTE: https://github.com/inveniosoftware/troubleshooting/issues/1
-        ":python_version=='2.7'": ["futures", "pathlib2"],
+        ":python_version=='2.7'": ["futures", "pathlib2", "flufl.lock==2.4.1"],
+        ":python_version>='3.0'": ["flufl.lock>=3.2"],
```

Comment:
Btw, we now need to add flufl to our https://github.com/conda-forge/dvc-feedstock/blob/master/recipe/meta.yaml . Not sure if it has conda package already. Probably not :slightly_frowning_face:

Reply:
There is a feedstock for `flufl.lock` - https://github.com/conda-forge/flufl.lock-feedstock

Reply:
Could you send a patch that would remove zc lockfile and add flufl to our conda reqs, please?

Reply:
Need a dvc release, which uses flufl locks first.

Reply:
@Suor , oh, right. I'll patch it myself on the next auto-generated PR then.